src/sass/style.scss

```
@@ -1,10 +1,10 @@
-@import "mixins", "navigation", "main", "footer";
+@import "common/mixins", "common/constants", "components/navigation", "components/main", "components/footer";
```

For a long amount of imports (maybe 4 or more?) I'd suggest spitting them into separate lines for clarity:

```suggestion
@import "common/mixins";
@import "common/constants";
@import "components/navigation";
@import "components/main";
@import "components/footer";
```